server/src/entity/Review.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  CreateDateColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+import { Account } from './Account';
+import { Restaurant } from './Restaurant';
+import { MenuItem } from './MenuItem';
+
+@Entity()
+export class Review extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @CreateDateColumn()
+  public dateReviewed: Date;
+
+  @Column({
+    default: 0,
+  })
+  public restaurantReviewScore: number;
+
+  @Column({
+    default: '',
+  })
+  public restaurantReviewComment: string;
+
+  @Column({
+    default: 0,
+  })
+  public itemReviewScore: number;
+
+  @Column({
+    default: '',
```

Comment:
why empty review comment?

Reply:
the person can review either item or restaurant. review is a general term